### PR TITLE
Pair programming

### PR DESCRIPTION
**Pull request by Joel-100**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,373 +1,404 @@
...
+            <input type="email" name="email" maxlength="30" placeholder="Email" />
+            <textarea name="message" maxlength="500" placeholder="Write your message here..."></textarea>
+            <button type="submit" class="btn">
+              <span>Get in touch</span>
+            </button>  
+          </form>
+        </div> 
+      </section>
+  </body>
+</html>
```
Comment (Munsa1) [inline index.html:399]: - [x] There is an overflow on your project which is causing the contact form to not be properly aligned in mobile view. Kindly fix it.

Your Design
image

Figma Design
image

- [x] Notice the gray space around your design

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Joel-100, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Joel-100 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the requested changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Joel-100** [comment]
Thanks!

**Pull request merged**